services/device-registry/src/main/java/org/eclipse/hono/deviceregistry/impl/FileBasedCredentialsService.java

```diff
@@ -10,7 +10,9 @@
  * Bosch Software Innovations GmbH - initial creation
  */
 
-package org.eclipse.hono.service.credentials.impl;
+package org.eclipse.hono.deviceregistry.impl;
```

Comment:
I moved it to org.eclipse.hono.deviceregistry as well. 

As soon as the registration API is contained (next PR) the other option is better IMHO:
- service.credentials.impl for FileBasedCredentialsService.java
- service.registration.impl for FileBasedRegistrationService.java
WDYT?